src/App.css

```diff
@@ -53,4 +53,44 @@ li{
 
 .article-card img{
   width:7rem;
+}
+
+.article-full{
+  display: flex;
+  flex-direction: column;
+  
+  background-color:rgb(245, 245, 245);
+  padding-left:1rem;
+  padding-right:1rem;
+  
+  border:1px solid black;
+  
+}
+
+.article-full h2{
+  text-decoration: underline;
+  text-transform: capitalize;
+}
+
+.article-full img{
+  margin-top: 0.5rem;
+  width:100%;
+  align-self: center;
+}
+
+.time-date{
+  font-size: 0.7rem;
+  font-weight: 500;
+}
+
+.comments-comp{
+  margin-top: 1rem;
+  border: 2px solid orange;
+  background-color: rgb(245, 245, 245);;
+}
+
+.votes-comp{
+  margin-top: 1rem;
+  border: 2px solid black;
+  background-color: rgb(245, 245, 245);;
```

Comment:
Currently the article list is fine, but now the text is not visible when I open a single article. Some css might need to be applied to that separately again.